src/Client/pages/Login.fs

```diff
@@ -41,22 +41,23 @@ let authUser (login:Login) =
 
         let body = toJson login
 
-        let props = 
+        let props =
             [ RequestProperties.Method HttpMethod.POST
+              RequestProperties.Credentials RequestCredentials.Sameorigin
```

Comment:
what's this doing?

Reply:
This would add cookies for `fetch` API,  see: https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch
![image](https://user-images.githubusercontent.com/5233940/37646673-3aca148e-2c65-11e8-90d7-2c53f764e1fd.png)

But I guess we don't need it here, I'll remove it.